Kernel/Arch/x86/common/Interrupts.cpp

```diff
@@ -640,6 +640,54 @@ UNMAP_AFTER_INIT void idt_init()
         register_interrupt_handler(i, unimp_trap);
 
     dbgln("Initializing unhandled interrupt handlers");
+    register_interrupt_handler(0x20, interrupt_32_asm_entry);
+    register_interrupt_handler(0x21, interrupt_33_asm_entry);
```

Comment:
If we're making these valid interrupts, the for loop above should be removed.

Reply:
Not really, we still need this from 0x11 to 0x1f. I will fix this now.